RecoParticleFlow/PFProducer/python/particleFlowEGamma_cff.py

```diff
@@ -23,14 +27,29 @@
 particleBasedIsolationTmp.valueMapElePFblockIso = cms.string("gedGsfElectronsTmp")
 
 egmPhotonIsolationCITK = _egmPhotonIsolationAOD.clone()
+egmElectronIsolationCITK = _egmElectronIsolationCITK.clone()
+egmElectronIsolationPileUpCITK = _egmElectronIsolationCITKPileUp.clone()
 
-for iPSet in IsoConeDefinitionsTmp:
+#change particleBasedIsolation object to tmp
+for iPSet in IsoConeDefinitionsPhotonsTmp:
```

Comment:
I tried to do that but I got the error message that VPSet doesn't have a clone() defined.